feat(starfish): Adds an initial web vitals/page loads module to starfish side nav

edwardgou-sentry opened the pull request:
Adds an experimental Browser Starfish page focused on displaying webvitals and page load performance scores

Commit:
Adds an initial web vitals/page loads module to starfish side nav

Commit:
Adds an initial web vitals/page loads module to starfish side nav

Commit:
Merge branch 'master' of github.com:getsentry/sentry into egou/feat/starfish/browser/web-vitals-module

Commit:
Merge branch 'egou/feat/starfish/browser/web-vitals-module' of github.com:getsentry/sentry into egou/feat/starfish/browser/web-vitals-module

DominikB2014 commented on static/app/views/performance/browser/webVitals/pagePerformanceTables.tsx:
```diff
@@ -0,0 +1,202 @@
+    },
+    pageFilters.selection
+  );
+  const {data, isLoading} = useDiscoverQuery({
+    eventView,
+    limit: 50,
+    location,
+    orgSlug: organization.slug,
+    options: {
+      enabled: pageFilters.isReady,
+      refetchOnWindowFocus: false,
+    },
+  });
```
Long term if bot to have a seperate hook for this data so that it cleans it up and we can introduce some better typing.

edwardgou-sentry committed:
feat(starfish): Adds individual webvital drawers to browser starfish webvitals module (#56205)

Adds drawers to webvitals, displaying worst pageloads on the respective
webvital.

edwardgou-sentry committed:
Update static/app/views/performance/browser/webVitals/meterBar.tsx

Co-authored-by: Dominik Buszowiecki <44422760+DominikB2014@users.noreply.github.com>

Commit:
fix qs import
